### ISSUE TITLE
Build error for f25 image in contrib

### ISSUE THREAD

**Issue by garethr** [puppetlabs/libral#48]
Running `make f25` in the `contrib/docker` directory fails with the following error:

```
[ 70%] Build mruby
/bin/sh: rake: command not found
make[2]: *** [mruby/host/bin/mruby] Error 127
mruby/CMakeFiles/mruby.dir/build.make:345: recipe for target 'mruby/host/bin/mruby' failed
make[1]: *** [mruby/CMakeFiles/mruby.dir/all] Error 2
CMakeFiles/Makefile2:389: recipe for target 'mruby/CMakeFiles/mruby.dir/all' failed
make: *** [all] Error 2
Makefile:140: recipe for target 'all' failed
The command '/bin/sh -c git -C /usr/src clone https://github.com/puppetlabs/libral &&     mkdir -p /usr/src/libral/build &&     cd /usr/src/libral/build &&     cmake .. &&     make all install' returned a non-zero code: 2
make: *** [f25] Error 2
```

**Comment by garethr** [comment]
Looks like the Dockerfile is missing `ruby rake bison` from the HACKING instructions. PR incoming if that works.

**Comment by garethr** [comment]
Fix up in https://github.com/puppetlabs/libral/pull/49

**Comment by lutter** [comment]
Fixed by PR #49
